XTS/WSCF/classes/com/arjuna/mwlabs/wscf/model/sagas/arjunacore/BACoordinator.java

```diff
@@ -428,7 +434,7 @@ public CoordinatorId identifier () throws SystemException
 
     public String type ()
     {
-        return "/StateManager/BasicAction/AtomicAction/Sagas/BACoordinator";
+        return TYPE;
```

Comment:
I see where you use getType() in the recovery module. I wonder if you should encode the type in the participant record so it will know what the coordinator type is rather than hard code the reference here.

Maybe merge this, raise a Jira and add a docs note?